cpp/arcticdb/python/python_module.cpp

```diff
@@ -271,10 +272,27 @@ PYBIND11_MODULE(arcticdb_ext, m) {
     arcticdb::async::register_bindings(m);
     arcticdb::codec::register_bindings(m);
     arcticdb::column_store::register_bindings(m);
-    arcticdb::storage::apy::register_bindings(m, base_exception);
+
+    auto storage_submodule = m.def_submodule("storage", "Segment storage implementation apis");
+    auto no_data_found_exception = py::register_exception<arcticdb::storage::NoDataFoundException>(
+            storage_submodule, "NoDataFoundException", base_exception.ptr());
+    arcticdb::storage::apy::register_bindings(storage_submodule, base_exception);
+
     arcticdb::stream::register_bindings(m);
     arcticdb::toolbox::apy::register_bindings(m);
-    arcticdb::version_store::register_bindings(m, base_exception);
+
+    m.def("get_version_string", &arcticdb::get_arcticdb_version_string);
```

Comment:
I know @qc00 has been working on this but I'm surprised to see this in this PR?

Reply:
Okay, you've removed from version python_bindings.

Reply:
Should just remove this function then. It does nothing.